### DISCUSSION TITLE
Can't import functions from helper function

### DISCUSSION THREAD

**Discussion by D1Asif** [mrdbourke/pytorch-deep-learning#579] [Q&A] [answered]
Getting the following error (Image attached). Wrote the exact code in (Chapter 2, video 71, minute 10:53:53)

![image](https://github.com/mrdbourke/pytorch-deep-learning/assets/76836400/6115a021-5326-4aea-b7e5-1b0bc3a17a4d)

**Comment by D1Asif** [comment, marked as answer]
Solved it on my own. It looks like if I restart and try to import the functions from a separate Colab cell, the thing works!

**Reply by mrdbourke** [reply]
Good to hear you got it solved!

A quick note: if you go away from Google Colab, the `helper_functions.py` file will delete itself (as Google Colab clears all storage after a while).

So you will need to redownload it each time you take a large break and come back to Google Colab.
